_includes/assets/sass/_variables.scss

```diff
@@ -2,6 +2,11 @@
 VARIABLES
 ========================================================================== */
 
+@import "node_modules/govuk-frontend/govuk/tools/px-to-rem";
```

Comment:
Is this node module already included in the repo? Is that why this PR is only SCSS and CSS files without the govuk-frontend module needing to be added, i.e. the function already exists in the repo, just not used until this commit?

Reply:
@joesb -  govuk-frontend already exists, pulled in after it was added as a dependancy to package.json file as part of work done on the project set up branch last week. Yep exactly, just not used yet 👍